test/e2e/apimachinery/namespace.go

```diff
@@ -21,17 +21,20 @@ import (
 	"strings"
 	"sync"
 	"time"
+	"encoding/json"
```

Comment:
need to put this line(`"encoding/json"`) to line 20 before `"fmt"` to pass gofmt check.

Reply:
Updated in 9ed55bc5203